manager/integration/deploy/test.yaml

```diff
@@ -40,6 +40,12 @@ rules:
 - apiGroups: ["longhorn.io"]
   resources: ["supportbundles", "supportbundles/status"]
   verbs: ["*"]
+- apiGroups: ["source.toolkit.fluxcd.io"]
+  resources: ["helmrepositories"]
+  verbs: ["*"]
+- apiGroups: ["helm.toolkit.fluxcd.io"]
+  resources: ["helmreleases"]
+  verbs: ["*"]
```

Comment:
We can just use cluster admin role, so we don't need to worry about the permission later for any kind of tests.

This is another topic, so can be tackled in another ticket.

Reply:
Refined to use `ClusterRole cluster-admin`.